### DISCUSSION TITLE
Works fine with every other apps except fps games

### DISCUSSION THREAD

**Discussion by mrpiezo** [OpenTabletDriver/OpenTabletDriver#1002] [Q&A]
Hey i have a Huion WH1409 V2 tablet which is supported by OpenTabletDriver. I can use the driver with no problem in Relative Mode. But whenever i open a fps game such as Valorant i can not move my camera in game. It still registers the click function, also its works in game settings and lobby. I tried changing the windowed resolution from fullscreen also tried to change the Relative Mode to Absolute Mode. Anyone know what is the problem?

**Comment by InfinityGhost** [comment]
Valorant's anticheat blocks the SendInput API we use for cursor positioning. A workaround for this is to use the [VMultiMode plugin](https://github.com/X9VoiD/VoiDPlugins/wiki/VMultiMode) which is installable from the plugin manager. As it requires installing a kernel mode dummy driver, it bypasses using the SendInput API and the anticheat can't differ it from a real input source.

**Reply by mrpiezo** [reply]
how do i install kernel mode dummy driver? https://github.com/jserv/dummy-driver this is the only link i found but couldnt understand how to install it

**Reply by InfinityGhost** [reply]
https://github.com/X9VoiD/vmulti-bin/releases/tag/v1.0

**Reply by mrpiezo** [reply]
vmulti option works now, i can move my camera in game. But this time i cannot use click function, neither in game or desktop. Also pen buttons doesn't work too. Is there any solution?

**Reply by InfinityGhost** [reply]
Did you bind those buttons within the UI?

**Reply by mrpiezo** [reply]
Yes i did. In normal relative mode all buttons and click function works but i cannot move my camera in game, with vmulti relative mode option my pen just works for looking around, even in desktop i cannot click anything. I tried connecting my tablet with cable and with usb bluetooth.

**Reply by X9VoiD** [reply]
Make sure you followed the instructions from [VMultiMode Setup](https://github.com/X9VoiD/VoiDPlugins/wiki/VMultiMode#setup), and that you are using VMultiMode's [VMulti](https://github.com/X9VoiD/vmulti-bin/releases/tag/v1.0) and not any other implementations of VMulti (hawku/devocub/gaomon/huion).

**Reply by mrpiezo** [reply]
Sorry for the late reply, yes looks like i missed some parts now the buttons works. But this time my keyboard and click function not working together. I mean i cannot click whenever i press any buttons on my keyboard, only happening in vmulti mode. Is it normal?

**Reply by mrpiezo** [reply]
Apparently its only happening when im using left function, its not happening when i switch to middle or right options.

**Reply by widips** [reply]
have you solve these problems. I'm having the same problems here with my huion tablet

**Reply by mrpiezo** [reply]
Not the last one, but you can workaround the problem like this; 
Go to your mouse settings on windows, change primary button left to right.
Then go to in game settings and change shoot button to right click.

Also if you want to use scope or nearsight;
Change your pen button in opentabletdriver to middle mouse button or any other button that you aren't using expect left click.
And of course make it same in game settings too.

**Reply by iamkiubi** [reply]
> Yes i did. In normal relative mode all buttons and click function works but i cannot move my camera in game, with vmulti relative mode option my pen just works for looking around, even in desktop i cannot click anything. I tried connecting my tablet with cable and with usb bluetooth.

How did you get your camera to work? I can only get my cursor moving with Valorant open using Windows Ink but the camera doesn't work at all nor can I shoot. I have VMulti installed but it doesn't work for me.

**Comment by okluis99** [comment]
i can shoot in game but i cant look around can someone please help me fix this ive been trying for so long and ive done everything in order
i have a Wacom CTL-4100

**Reply by X9VoiD** [reply]
Make sure you're using relative output mode, and if the game is using anticheat then use VMulti relative mode.

**Reply by okluis99** [reply]
i am :(

**Reply by okluis99** [reply]
wait is there supposed to be VMulti relative mode or windows ink relative mode

**Reply by jamesbt365** [reply]
vmulti relative and you are literally seeing all the things valorant does to stop you. Blame them not us, nothing can bypass this.

**Comment by gerst2** [comment]
I have the same issue but when i get close to the drawing tablet it spins me around like crazy

**Reply by X9VoiD** [reply]
Use relative output modes.

**Comment by gonX** [comment]
I believe this discussion has most of the information necessary to get FPS games to pick up tablet input. A relative output mode should be used.

If you're having problems with Valorant specifically please see our Wiki: https://opentabletdriver.net/Wiki/FAQ/WindowsAppSpecific#valorant
